src/styles/sass/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  padding: 16px;
+  width: 200px;
+  height: 408px;
```

Comment:
don't hardcode height - it should depend on content (you just have to add correct margin-bottom for image)